RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -1929,34 +1947,21 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
 
       if (!mutrk) continue; //PH This should not be needed!
```

Comment:
I am not sure why, but the muonTracks array doesn't seem to contain all the muon tracks (you can compare "nmu" and "nmu_filled"). I would suggest having for the conversion of AOD an approach similar to what is done in the Central Barrel, eg:
Int_t ntrk = fVEvent->GetNumberOfTracks();
[...]
    for (Int_t imu = 0; imu < ntrk; ++imu) {
        AliVTrack * aodVTrack = fAOD->GetTrack(imu);
        AliAODTrack * mutrk = dynamic_cast<AliAODTrack *>(aodVTrack);
        if (!mutrk) continue; // Should not happen
        if (! (mutrk->IsMuonTrack() ) ) continue; // Skip if not a muon track
       [...]
}

Reply:
The purpose of this PR is to create the forward track table and convert RUN2 ESD to RUN3 AOD. The Run2 AOD -> Run3 AOD converter have problems and fixing it is beyond the scope of this PR. I invite users of the Muon Run2 AOD -> Run3 AOD converter to fix it after this PR is merged.

By the way, the forward track table created in this PR is necessary to convert RUN2 global muon tracks for RUN3 AOD.

Reply:
I agree, lets merge this PR. I am fine with the latest changes from Maxime.